### ISSUE TITLE
Add viewBox attribute to svg element for proper scaling

### ISSUE THREAD

**Issue by vorg** [runemadsen/rune.js#30]
Consider the following code:

```javascript
var r = new Rune({ width: 600, height: 800 })
r.el.style.width = "300px"
r.el.style.height = "400px"
```

Instead of scaling the svg it will be cropped. To scale it properly we need to add viewBox propery

```javascrip
r.el.setAttribute('viewBox', `0 0 600 800`)
```

<img width="584" alt="screen shot 2017-08-12 at 00 43 24" src="https://user-images.githubusercontent.com/171001/29235503-49029b46-7ef7-11e7-8bef-15f24fc8de9e.png">

**Comment by runemadsen** [comment]
Good idea. Adding to TODO. Keeping issue open until it's implemented.

**Comment by runemadsen** [comment]
Just fixed this in the upcoming `1.1.1`.
